src/hardware/vga_draw.cpp

```diff
@@ -1621,25 +1621,37 @@ void VGA_SetupDrawing(uint32_t /*val*/)
 	case M_HERC_GFX:
 		vga.draw.blocks=width*2;
 		width*=16;
+
+		assert(width > 0);
+		assert(height > 0);
 		aspect_ratio = (static_cast<double>(width) / static_cast<double>(height)) *
 		               (3.0 / 4.0);
+
 		VGA_DrawLine = VGA_Draw_1BPP_Line;
 		break;
 	case M_TANDY2:
 		aspect_ratio = 1.2;
 		doubleheight=true;
-		if (machine==MCH_PCJR) doublewidth=(vga.tandy.gfx_control & 0x8)==0x00;
-		else doublewidth=(vga.tandy.mode_control & 0x10)==0;
+		if (machine == MCH_PCJR)
+			doublewidth = (vga.tandy.gfx_control & 0x8 == 0x0);
```

Comment:
What about binary literals for flag/bit twiddling values? My brain can more easily parse that, especially when checking against e.g. a tech ref manual.

Reply:
Created a new bit-table (most bit are the same between the Tandy and PCjr). 
Will be a perfect use of bit-literals. 
(My brain also doesn't magically convert things like 0x13 into a bank of high-and-low bits).